### ISSUE TITLE
Sort file list in commit dialog in modification order

### ISSUE THREAD

**Issue by AndersBillLinden** [rabbitvcs/rabbitvcs#196]
I need one more column in rabbitvcs commit. One that shows modification date.
Then I will be able to see which changes that are the most recent and select those when committing.

**Comment by czanoni-gh** [comment]
Yes please.   I've been looking for this since I started using RabbitVCS 4 months ago...
